### DISCUSSION TITLE
Change the breakpoints

### DISCUSSION THREAD

**Discussion by joomlaplates-de** [templaza/astroid-framework#243] [Q&A]
Hi all, 
I just try to change the breakpoints that the main menu is longer visible.
Anyone a tip, because this is not working
```
$grid-breakpoints: (
  // Extra small screen / phone
  xs: 0,
  // Small screen / phone
  sm: 576px,
  // Medium screen / tablet
  md: 768px,
  // Large screen / desktop
  lg: 992px,
  // Extra large screen / wide desktop
  xl: 1200px
);

$container-max-widths: (
  sm: 540px,
  md: 720px,
  lg: 960px,
  xl: 1140px
);
```
Thanks Peter

**Comment by sonvnn** [comment]
Hello,

Let's try to use Basic > Layout Settings > Theme Width. It will override container max widths
![Screen Shot 2022-11-10 at 15 47 03](https://user-images.githubusercontent.com/3204103/201043713-b264d74f-baeb-4bee-becd-682d46b0a309.png)


Best Regards,
Sonny

**Reply by joomlaplates-de** [reply]
Hi Sonny, yes I know that.
But I want to have a width of 1200px and want that the menu is showing until 768px and not collapsing by 992px.
